src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeCleanup/CodeCleanupOptions.cs

```diff
@@ -8,12 +8,12 @@
 using Microsoft.CodeAnalysis.Formatting;
 using Microsoft.CodeAnalysis.Simplification;
 using Microsoft.CodeAnalysis.AddImport;
-using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.CodeActions;
 using Roslyn.Utilities;
 using Microsoft.CodeAnalysis.Diagnostics;
 
 #if !CODE_STYLE
+using Microsoft.CodeAnalysis.Host;
```

Comment:
needed as the old code was in CodeStyle/Worskapces.  It previously saw WeaklyCachedValueSource in 'Host'. But now that that moved to Workspace-only, this namespace is now empty and we were getting errors when compiling CODE_STYLE.